app/views/layouts/temp_home.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
His reasoning seems a bit vague and personal. I like HAML but its main problem is that libraries like Polymer and Angular make a lot of use of inline templates that are difficult to support in something other than raw HTML.